Templates/Templates/AutoGenerateProtocol.stencil

```diff
@@ -0,0 +1,32 @@
+
+//: Do not change this code as it is autogenerated every time you build.
+//: You can change the code in `../StencilTemplatesForSourcery/Application/AutoGenerateProtocol
+
+// MARK: - AutoGenerateProtocol
+//: From all Types implementing this protocol Sourcery adds:
+//: - public/internal variables // private variables are ignored
+//: - public/internal methods (skips initializers)
+//: - initializers marked with annotation // sourcery:includeInitInProtocol
+//: - of the above it does not add it if  // sourcery:skipProtocol
+//: ---
+
+{% for type in types.implementing.AutoGenerateProtocol %}
+// MARK: - {{ type.name }} AutoGenerated protocol
+protocol {{ type.name }}Protocol: AutoMockable {
```

Comment:
I guess your main intention for this template is to generate protocols for mocks, but I think this conformance should be something that should be added manually.